context.go

```diff
@@ -22,8 +22,9 @@ import (
 )
 
 const (
-	flagSampled = byte(1)
-	flagDebug   = byte(2)
+	flagSampled  = byte(1)
+	flagDebug    = byte(2)
+	flagFirehose = byte(4)
```

Comment:
Maybe we could define is as the 4th bit (`byte(8)`), not the 3rd. Zipkin supports "undecided" sampling decision, which we might need to implement in the future (for edge proxy use cases), and it's nicer to group sampling bits together.

Reply:
Also, should we add a bit of docs here to what firehose means?

Reply:
Yes, we can use the fourth bit. 

I think we should add docs on a centralized place like the website would be better, or we would be repeating the semantic meaning in every client. (I don't think we have docs for debug flag in the clients either).